satellite/satellitedb/overlaycache.go

```diff
@@ -987,6 +987,96 @@ func (cache *overlaycache) UpdatePieceCounts(ctx context.Context, pieceCounts ma
 	return Error.Wrap(err)
 }
 
+// GetExitingNodes returns nodes in exiting status.
+func (cache *overlaycache) GetExitingNodes(ctx context.Context) (exitingNodes storj.NodeIDList, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	rows, err := cache.db.Query(cache.db.Rebind(`
+		SELECT id FROM nodes
+		WHERE exit_loop_completed_at IS NULL
+		AND exit_initiated_at IS NOT NULL
```

Comment:
I think this should be
```
		WHERE exit_finished_at IS NULL
		AND exit_initiated_at IS NOT NULL
```

Reply:
okay I can change it- I was using what the doc said though. should I update that too? https://github.com/storj/storj/blob/master/docs/blueprints/storagenode-graceful-exit/pieces.md#design

Reply:
Sorry. I misunderstood this methods purpose.  I think we need 2 methods.
1 that returns exiting nodes that have initiated, but are not finished.
another that returns exiting nodes that haven't completed the metainfo loop yet.